Sources/UI/Chat/Chat View Controller/ChatViewController+MessageActions.swift

```diff
@@ -311,11 +311,14 @@ extension ChatViewController {
             return nil
         }
         
+        guard let cell = tableView.cellForRow(at: indexPath),
+            let message = self.presenter?.items[safe: indexPath.row]?.message else {
```

Comment:
Can you make sure this line aligns with the line above?

Reply:
👍